state/modelsummaries_test.go

```diff
@@ -1,514 +0,0 @@
-// Copyright 2017 Canonical Ltd.
-// Licensed under the AGPLv3, see LICENCE file for details.
-
-package state_test
-
-import (
-	"fmt"
-	"sort"
-	"time"
-
-	"github.com/juju/mgo/v3/bson"
-	"github.com/juju/names/v5"
-	jc "github.com/juju/testing/checkers"
-	gc "gopkg.in/check.v1"
-
-	"github.com/juju/juju/core/arch"
-	"github.com/juju/juju/core/instance"
-	"github.com/juju/juju/core/permission"
-	"github.com/juju/juju/core/status"
-	"github.com/juju/juju/internal/storage"
-	"github.com/juju/juju/internal/testing"
-	"github.com/juju/juju/internal/testing/factory"
-	"github.com/juju/juju/internal/uuid"
-	"github.com/juju/juju/state"
-)
-
-type ModelSummariesSuite struct {
```

Comment:
You might as well implement the other parts of the summary in this patch, and ensure they have this coverage.

We have the tables for machines/apps/units; you just need to pull the counts.

Reply:
Are these tables not on the model db rather than the controller db? In mongo to get this information it went through the collections for each model. I guess after [recent discussions](https://github.com/juju/juju/pull/17841#pullrequestreview-2215011195) we shouldn't be going into other models to get this info. At what level and how would you recommend populating this?